src/pages/Summary.jsx

```diff
@@ -46,9 +48,19 @@ const Summary = () => {
 
             if (newData) {
                 if (Capacitor.getPlatform() !== "web") LocalNotifications.clearDeliveredNotifications();
+                let keys = localStorage.getItem("keys");
```

Comment:
should probably make this a function and call it in summary and journal. actually, wait, this can replace auth likely! this definitively tells you whether a user has auth or not. easy as that.